src/Microsoft.PowerPlatform.PowerApps.Persistence/Models/ControlPropertiesCollection.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.Models;
+
+public class ControlPropertiesCollection : IReadOnlyDictionary<string, ControlPropertyValue>
```

Comment:
I'm not sure if there are any benefits to making our own collection type instead of just using `IReadOnlyDictionary<string, ControlPropertyValue>`.

Anything that I'm not seeing which a custom collection would help wtih?

Reply:
@anpetroc suggested it this morning. Honestly, I'm not super opposed to it, but I'm fine with reverting back to `IReadOnlyDictionary` for now

Reply:
Hmm.   I guess it does give a nicer target type to `ControlPropertiesCollectionConverter` than just looking for a dictionary.

Reply:
`IEnumerable<KeyValuePair<string, ControlPropertyValue>>` would have worked too in the converter btw